src/planscape/conditions/raster_utils_test.py

```
@@ -12,6 +14,129 @@
+
+    def test_returns_none_for_no_geo(self):
+        foo_raster = _create_raster(self, 4, 4, (1, 2, 3, 4,
+                                                 5, 6, 7, 8,
+                                                 9, 10, 11, 12,
+                                                 13, 14, 15, 16))
+        _create_condition_raster(foo_raster, "foo")
+        score = compute_condition_score_from_raster(None, "foo")
+        self.assertIsNone(score)
+
+    def test_fails_wrong_srid(self):
+        polygon = Polygon(
+            ((-120, 40),
+             (-120, 41),
+             (-121, 41),
+             (-121, 40),
+             (-120, 40)))
+        geo = MultiPolygon(polygon)
+        geo.srid = 4269
+        with self.assertRaises(Exception) as context:
+            compute_condition_score_from_raster(geo, "foo")
+        self.assertEqual(
+            str(context.exception), "geometry SRID is 4269 (expected 9822)")
+
+    def test_returns_none_for_no_intersection(self):
+        geo = _create_geo(self, 7, 10, 0, 1)
+        foo_raster = _create_raster(self, 4, 4, (1, 2, 3, 4,
+                                                 5, 6, 7, 8,
+                                                 9, 10, 11, 12,
+                                                 13, 14, 15, 16))
+        _create_condition_raster(foo_raster, "foo")
+        score = compute_condition_score_from_raster(geo, "foo")
+        self.assertIsNone(score)
+
+    def test_returns_none_for_no_raster(self):
+        geo = _create_geo(self, 0, 3, 0, 1)
+        foo_raster = _create_raster(self, 4, 4, (1, 2, 3, 4,
+                                                 5, 6, 7, 8,
+                                                 9, 10, 11, 12,
+                                                 13, 14, 15, 16))
+        _create_condition_raster(foo_raster, "foo")
+        score = compute_condition_score_from_raster(
+            geo, "nonexistent_raster_name")
+        self.assertIsNone(score)
```

maybe we should return an error for this?

---

Done.